cms/envs/test.py

```diff
@@ -275,86 +177,52 @@
 )
 VIDEO_IMAGE_DEFAULT_FILENAME = "default_video_image.png"
 
-########################## VIDEO TRANSCRIPTS STORAGE ############################
-VIDEO_TRANSCRIPTS_SETTINGS = dict(
-    VIDEO_TRANSCRIPTS_MAX_BYTES=3 * 1024 * 1024,  # 3 MB
-    STORAGE_KWARGS=dict(
-        location=MEDIA_ROOT,
-        base_url=MEDIA_URL,
-    ),
-    DIRECTORY_PREFIX="video-transcripts/",
-)
-
-####################### Plugin Settings ##########################
+############################## Authentication ##############################
 
-# pylint: disable=wrong-import-position, wrong-import-order
-from edx_django_utils.plugins import add_plugins
+JWT_AUTH.update(jwt_jwks_values)
 
-# pylint: disable=wrong-import-position, wrong-import-order
-from openedx.core.djangoapps.plugins.constants import ProjectType, SettingsType
+####################### Plugin Settings ##########################
 
 add_plugins(__name__, ProjectType.CMS, SettingsType.TEST)
 
 ########################## Derive Any Derived Settings  #######################
 
 derive_settings(__name__)
 
-############### Settings for edx-rbac  ###############
-SYSTEM_WIDE_ROLE_CLASSES = os.environ.get("SYSTEM_WIDE_ROLE_CLASSES", [])
-
-DEFAULT_MOBILE_AVAILABLE = True
+################################ Templates #################################
 
-PROCTORING_SETTINGS = {}
-
-# Used in edx-proctoring for ID generation in lieu of SECRET_KEY - dummy value
-# (ref MST-637)
-PROCTORING_USER_OBFUSCATION_KEY = "85920908f28904ed733fe576320db18cabd7b6cd"
+# For the "preview" template engine, the CMS uses the same dirs as the LMS. Here we mutate
+# the DIRS list to match the MAKO_TEMPLATE_DIRS_BASE list produced by lms.envs.test.
+preview_template = next(d for d in TEMPLATES if d["NAME"] == "preview")
+preview_template['DIRS'].extend([
+    COMMON_ROOT / 'test' / 'templates',
+    COMMON_ROOT / 'test' / 'test_sites',
+    REPO_ROOT / 'openedx' / 'core' / 'djangolib' / 'tests' / 'templates',
+])
+for theme_dir in COMPREHENSIVE_THEME_DIRS:  # pylint: disable=not-an-iterable
+    preview_template['DIRS'].insert(0, theme_dir)
 
 ############### Settings for Django Rate limit #####################
 
 RATELIMIT_RATE = '2/m'
 
-##### LOGISTRATION RATE LIMIT SETTINGS #####
-LOGISTRATION_RATELIMIT_RATE = "5/5m"
-LOGISTRATION_PER_EMAIL_RATELIMIT_RATE = "6/5m"
-LOGISTRATION_API_RATELIMIT = "5/m"
-
-REGISTRATION_VALIDATION_RATELIMIT = "5/minute"
-REGISTRATION_RATELIMIT = "5/minute"
-OPTIONAL_FIELD_API_RATELIMIT = "5/m"
-
-RESET_PASSWORD_TOKEN_VALIDATE_API_RATELIMIT = "2/m"
-RESET_PASSWORD_API_RATELIMIT = "2/m"
-
-############### Settings for proctoring  ###############
-PROCTORING_USER_OBFUSCATION_KEY = "test_key"
```

Comment:
Just making a note that the new default in the common file is different but that's probably fine for this test key.